src/gpu.cpp

```diff
@@ -173,7 +173,7 @@ void GPUS::find_active_gpu() {
                 for (auto& pid : gpu->nvidia_pids()) {
                     if (pid == getpid()) {
                         gpu->is_active = true;
-                        SPDLOG_DEBUG("Active GPU Found: node_name: {}, pci_dev: {}", gpu->name, gpu->pci_dev);
```

Comment:
should be debug

Reply:
since find_active_gpu is nuked, i dont think this is necessary

Reply:
You're right :+1: